crates/re_renderer/src/renderer/point_cloud.rs

```diff
@@ -725,3 +725,47 @@ impl Renderer for PointCloudRenderer {
         Ok(())
     }
 }
+
+/// Determines the size for a "data texture", i.e. a texture that we use to store data
+/// that we'd otherwise store inside a buffer, but can't due to WebGL compatibility.
+///
+/// `max_texture_size` must be a power of two.
+///
+/// For convenience, the returned texture size has a width that is also a multiple of `wgpu::COPY_BYTES_PER_ROW_ALIGNMENT`
+/// for the given element size.
+/// This makes it a lot easier to copy data from a continuous buffer to the texture.
+/// If we wouldn't do that, we'd need to do a copy per row in some cases.
+fn data_texture_size(
```

Comment:
Is this assuming the texture is one-byte-per-texel or what?
or is the `element_size_in_bytes` the number of bytes per text, and depends on the texture format?